packages/experimental/Link/src/useLink.ts

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { useAsPressable, useKeyProps, useOnPressWithFocus, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { LinkProps, LinkState } from './Link.types';
+import { memoize } from '@fluentui-react-native/framework';
+import { AccessibilityState, Linking, Platform } from 'react-native';
+
+export const useLink = (props: LinkProps): LinkState => {
+  // attach the pressable state handlers
```

Comment:
I don't think this comment makes sense here, maybe where we call useAsPressable?